test/specs/parsing/tests/KindTest.tests.ts

```diff
@@ -46,18 +46,6 @@ describe('KindTest', () => {
 		chai.assert.isTrue(evaluateXPathToBoolean('self::text()', documentNode.documentElement.firstChild));
 	});
 
-	if (typeof DOMParser !== 'undefined') {
```

Comment:
Why was this removed? Was this not used?

Reply:
The check was not necessary

Reply:
These are the tests, there is never a DOMParser